server/server.go

```diff
@@ -88,6 +89,25 @@ func (s *Server) clientForRequest(req parseRequest) (protocol.ProtocolServiceCli
 	return newClient(req.ServerURL)
 }
 
+func (s *Server) LoadGist(ctx *gin.Context) {
```

Comment:
why do you need proxy?
Github allows to do cross origin requests to gist.githubusercontent.com

Reply:
That's a quite good approach. Since you're resuming this issue, feel free to proceed that way